content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -169,130 +169,159 @@ Once you have a Linux-based Kubernetes master node you are ready to choose a net
 
-1. Copy Kubernetes certificate
+  ```PowerShell
+    .\KubeCluster.ps1 -ConfigFile .\Kubeclustervxlan.json -install 
+  ```
+  where `-ConfigFile` points to the path of the Kubernetes configuration file.
 
-    Copy the Kubernetes certificate file `$HOME/.kube/config` [from the Linux controller](https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/creating-a-linux-master#collect-cluster-information) to this new `C:\k` directory on your Windows node.
+{{< note >}}
+In the example below, we are using overlay networking mode. This requires Windows Server version 2019 with [KB4489899](https://support.microsoft.com/help/4489899) and at least Kubernetes v1.14 or above. Users that cannot meet this requirement must use  `L2bridge` networking instead by selecting `bridge` as the [plugin](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/v1.15.0/Kubeclusterbridge.json#L18) in the configuration file.  
+{{< /note >}}
 
-    Tip: You can use tools such as [xcopy](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/xcopy), [WinSCP](https://winscp.net/eng/download.php), or this [PowerShell wrapper for WinSCP](https://www.powershellgallery.com/packages/WinSCP/5.13.2.0) to transfer the config file between nodes.
+  ![alt_text](../kubecluster.ps1-install.gif "KubeCluster.ps1 install output")
 
-1. Download Kubernetes binaries
 
-    To be able to run Kubernetes, you first need to download the `kubelet` and `kube-proxy` binaries. You download these from the Node Binaries links in the CHANGELOG.md file of the [latest releases](https://github.com/kubernetes/kubernetes/releases/). For example 'kubernetes-node-windows-amd64.tar.gz'. You may also optionally download `kubectl` to run on Windows which you can find under Client Binaries.
+On the Windows node you target, this step will:
+  1. Enable Windows Server containers role (and reboot)
+  1. Download and install the chosen container runtime
+  1. Download all needed container images
+  1. Download Kubernetes binaries and add them to the `$PATH` environment variable
+  1. Download CNI plugins based on the selection made in the Kubernetes Configuration file
+  1. (Optionally) Generate a new SSH key which is required to connect to the control-plane ("Master") node during joining
+      {{< note >}}
+      For the SSH key generation step, you also need to add the generated public SSH key to the `authorized_keys` file on your (Linux) control-plane node. You only need to do this once. The script prints out the steps you can follow to do this, at the end of its output.
+      {{< /note >}}
 
-    Use the [Expand-Archive](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.archive/expand-archive?view=powershell-6) PowerShell command to extract the archive and place the binaries into `C:\k`.
 
-#### Join the Windows node to the Flannel cluster
+#### Join the Windows Node to the Kubernetes cluster
```

Comment:
Before this, can we include a section that says that if you want to replace binaries or edit configs, you should do that post install and pre join

Reply:
Added this